lib/defaultTheme.js

```diff
@@ -5,6 +5,7 @@ const theme = {
   colors: {
     ...DefaultTheme.colors,
     primary: '#2165E3',
+    text: '#6D6D68',
```

Comment:
ok sorry I have probably misled you a bit on this. the default color in paper is black, but then you'll see that they'll make it lighter in the typography components. doing this here will make our text match this color, but the text elements created using paper typography components lighter than this. I hope this is clear now

Reply:
I've updated the typography components to use `color` module internally. Similar to how `react-native-paper` approached it https://github.com/callstack/react-native-paper/blob/main/src/components/Typography/StyledText.tsx#L16. Here's the commit https://github.com/nearform/optic-expo/pull/23/commits/1d392074239c7566db1f22bce50b35cf966167bc